custom_model_runner/datarobot_drum/drum/model_metadata.py

```diff
@@ -43,7 +43,7 @@
 PARAM_NAME_MAX_LENGTH = 64
 
 # Max length of a select value
-PARAM_SELECT_VALUE_MAX_LENGTH = 32
+PARAM_SELECT_VALUE_MAX_LENGTH = 1024
```

Comment:
@klichukb @yakov-g we use this in: `datarobot_drum.drum.model_metadata.read_model_metadata_yaml`  where we validate this in a trafaret in: `datarobot_drum.drum.model_metadata.validate_model_metadata_hyperparameter`  why do we do this?  Isn't this the responsibility of the custom task version ingest?

also, if I want to apply this change, do I then need to bump the drum version and update all of the public drop-in envs?   if so, how do I do this?

Reply:
Hey, I'm almost completely not familiar with model metadata, its validation and hyper parameters.
It all was developed by Jesse,   (4dhalibut - I forgot the name) and i think you contributed a lot into this.

Regarding release:
1. Create a pr bumping DRUM, merge, release DRUM - there is a pipeline.
2. Create a PR bumping DRUM in the envs. Env versions IDs will be bumped automatically.
Maybe you could do 1 and 2 in a single PR, not sure how tests are configured.
3. If you think envs should be published: build installer and then run it to install envs

https://datarobot.atlassian.net/wiki/spaces/RAPTOR/pages/1416691967/Install+drop-in+environments+on+staging+production+sts

Reply:
@scottp-ml do you know why we have this here?